app/views/text_components/_form.html.haml

```diff
@@ -8,7 +8,7 @@
         #triggers
           - if local_assigns[:create_trigger]
             = f.simple_fields_for :triggers do |trigger|
-              = render 'triggers/trigger_fields', f: trigger
+              = render 'triggers/trigger_fields', f: trigger, is_sub_form: true
```

Comment:
I'm using conditionals here again, because I have to change the view at two different points - is there a cleaner solution?